docs/references/studio/api/datasets.md

```diff
@@ -0,0 +1,257 @@
+
+### List Datasets
+
+**Description**: Retrieve a list of datasets with optional prefix filtering.
+
+**Endpoint**: `/datachain/datasets`
+
+**Method**: `GET`
+
+**Authentication**: Requires token with `read` access to `DATASETS` scope from [`datachain auth login`](../../../commands/auth/login.md)
+
+**Parameters**:
+
+- `prefix` (query, optional): Filter datasets by name prefix
+- `team_name` (query, required): Team identifier (automatically added by client)
+
+**Example Request**:
+```bash
+curl -X GET "https://studio.datachain.ai/api/datachain/datasets?prefix=training&team_name=my-team" \
+  -H "Authorization: token YOUR_STUDIO_TOKEN" \
+  -H "Content-Type: application/json"
+```
+
+**Example Response**:
+```json
+[
+  {
+    "attrs": ["training"],
+    "created_at": "2025-06-11T14:22:31.123456+00:00",
+    "created_by_id": 2,
+    "description": "Training dataset for computer vision",
+    "id": 1853,
+    "name": "training-images-v2",
+    "project": {
+      "created_at": "2025-06-20T04:34:24.083394+00:00",
+      "created_by_id": null,
+      "descr": "Computer Vision Project",
+      "id": 2,
+      "name": "cv-project",
+      "namespace": {
+        "created_at": "2025-06-20T04:34:24.074155+00:00",
+        "created_by_id": null,
+        "descr": "CV Team Namespace",
+        "id": 2,
+        "name": "cv-team",
+        "team_id": 1,
```

Comment:
Same, we use `team_name` in param, and return `team_id`. Should it be something like this, at least?
```json
        "team": {
            "id": 1,
            "name": "my-team"
        },
```